t/Statements/Expressions/Value/Literal.t

```diff
@@ -3,12 +3,77 @@ use warnings;
 use Guacamole::Test;
 use Test::More;
 
-# LitNumber
+# LitNumberDec
 parses('4');
 parses('44');
-parses('-4');
 parses('44.4');
-parses('-44.4');
+parses("4.");
+parses(".4");
+parses("0");
+parses("0.");
+parses(".0");
+
+parses(".0e1");
+parses("0.e1");
+parsent(".e1");
+
+parses("0._4");
+parses(".4_");
+parsent("._4");
```

Comment:
Related to the previous comment, I would add here some test cases for `Underbars` before the decimal dot.

Reply:
Good idea, thanks.

Reply:
By which I meant, very confusingly, tests for stuff such as `1_234_567`, `2_00_000` (or however the Indian thing with crores and lakhs is done), etc.